content/02.main-text.md

```diff
@@ -40,11 +42,13 @@ In other words, as long as institutions have not integrated accurate models of c
 
 In an ideal world, we would have frictionless workflows that make it easy to introduce credit and attribution to authors and beyond. In addition, workflows would allow seamless ways to promote preservation and discoverability of work.
 Tools are being developed that help address this issue, such a [Manubot](https://manubot.org/) @doi:10/c7np, a workflow and set up tools that promotes open and automated workflows for writing manuscripts. This commentary was generated using manubot.
-A great advantage of using Manubot was that is was designed to traccck author contributions and made the contributions more transparent.
+A great advantage of using Manubot was that is was designed to track author contributions and made the contributions more transparent.
 [**Add something here about using CRO? Manubot should be be able to track our contributions in the yaml file too?**]
+[**JC: I am not sure that it actually works, github functionality is only counting commits number, not importance, and manubot would only track work done during manuscript writing. To get something automated we would need something much more complicated, where (for example) data would be published with a list of contributor, that list being imported into the manuscript while linking the manuscript to the data.**]
 Manubot trackers author metadata in a yaml file, which can be transfered to other manuscripts, saving time with collecting author details each time you write a manuscript.
 [**Can it be imported into author submission systems too, so we don't have to fill out the authorship sections in journals? That would be amazing!**]
-Manubot incorporates a simple workflow for tracking citations, and automatcally creates a reference list, removing the need to use a reference manager.
+[**JC: very little portability of the yaml actually, not even to other system using yaml to enter author information. One would need a codec to translate the list from one format to another, see github.com/open-science-promoters/contibutor_manager**]
+Manubot incorporates a simple workflow for tracking citations, and automatcally creates a reference list, removing the need to use a reference manager. [**putting citation just with doi has also its disadvantage, I do prefer a solution using a bib file (produced by a reference manager) and an author-date call to the bib file, otherwise one always need to click to see what the publication actually is**]
 Manubot is more technical to use compared to more commonly used authoring tools like Microsoft Word or Google Docs. All of the  authors were required to have a GitHub account and some basic familiarity with using GitHub, and with writing in Markdown.
```

Comment:
noting that the comment above is from Julien. (Thanks for the comment. Do you want to incorporate this into the text?